tools/unitctl/unit-openapi/README.md

```diff
@@ -409,3 +409,4 @@ cargo doc --open
 ## Author
 
 unit-owner@nginx.org
+
```

Comment:
What is it with this file!?

I thought we'd left that trailing newline in and then ignored it via a git attribute?

Reply:
That was actually my auto-ensure-trailing-newline func that comes with VS Code, but I actually had it in source to test the actual build (as it needs to be triggered by changes in that folder).